test/e2e/specs/tracks/tracks__lohp-to-signup-events.ts

```diff
@@ -0,0 +1,90 @@
+/**
+ * @group calypso-release
+ */
+
+import { DataHelper, TracksEventManager } from '@automattic/calypso-e2e';
+import { Page, Browser } from 'playwright';
+
+declare const browser: Browser;
+
+describe( DataHelper.createSuiteTitle( 'Verify Tracks events starting at LOHP' ), function () {
+	let page: Page;
+	let tracksEventManager: TracksEventManager;
+	const lohpUrl: string = 'https://wordpress.com/?flags=a8c-analytics.on';
+
+	beforeEach( async () => {
+		page = await browser.newPage();
+		tracksEventManager = new TracksEventManager( page );
+		tracksEventManager.allowTestsToFireEvents();
+	} );
+
+	afterEach( async () => {
+		await page.close();
+	} );
+
+	it( 'Loading LOHP fires wpcom_page_view event', async function () {
+		// Set up listener for wpcom_page_view event
+		const didEventFirePromise = tracksEventManager.didEventFire( 'wpcom_page_view' );
+
+		// Navigate to LOHP
+		await tracksEventManager.navigateToUrl( lohpUrl );
+
+		// Expect the event to fire
+		expect( await didEventFirePromise ).toBe( true );
+	} );
+
+	it( 'Clicking link on LOHP fires wpcom_link_click', async function () {
+		// Navigate to LOHP
+		await tracksEventManager.navigateToUrl( lohpUrl );
+
+		// Set up listener for wpcom_link_click event
+		const didEventFirePromise = tracksEventManager.didEventFire( 'wpcom_link_click' );
+
+		// Click first signup link to go to signup page
+		await page.click( 'a[href*="https://wordpress.com/start"]' );
```

Comment:
It's generally [recommended to use user-facing attributes](https://playwright.dev/docs/locators#:~:text=To%20make%20tests%20resilient%2C%20we%20recommend%20prioritizing%20user%2Dfacing%20attributes) for locators, which in this case would be

```js
await page.getByRole( 'menuitem', { name: 'Get Started' } ).click()
```

Having said that, if you think the URL is more resilient in this case, let's go with it. My only nitpick here would be to use the prefix matcher (`^=`) instead of the global one (`*=`):

```js
await page.locator( 'a[href^="https://wordpress.com/start"]' ).click();
```

Reply:
I think using the URL might be more resilient in this case. Given the changes we regularly make on our landing pages, we could easily end up change the button text. The route to the signup page is likely a bit more stable though.

I'll update it to use the prefix matcher. 👍